test/specs/login/Login.spec.js

```diff
@@ -0,0 +1,25 @@
+// login.spec.js
+var expect = require('chai').expect;

```

Comment:
You can move this expect in the global hooks
please review again
http://webdriver.io/guide/testrunner/frameworks.html